examples/util.ts

```diff
@@ -7,6 +7,7 @@
  */ /** */
 
 import { TRANSACTION_VERSION } from '@polkadot/types/extrinsic/v4/Extrinsic';
+import fetch from 'node-fetch';
```

Comment:
Why does this get imported in this PR but not used?

Reply:
It is used on line 22. I can't figure out exactly why it did not need to be imported before, but while trying out the example it was failing after bumping deps. In modern web browsers fetch is automatically included in the global scope while node does not include it.